styleguide/components/styleguide/styleguide.css

```diff
@@ -1,6 +1,10 @@
 @import "../../../src/styles/colors-v2.css";
 @import "../../../src/styles/spacing.css";
 
+:global html {
```

Comment:
I couldn't figure out a way to do with in the `styleguide.config.js` file. Supposedly, the `styles` key-value should take something along the lines of:

```
'@global html': { scrollPaddingTop: ... }
```

But it didn't work. I think for now this is fine and maybe later we can find more info on how to do better global styling in react-styleguidist.